test/Interop/Cxx/operators/member-inline-module-interface.swift

```diff
@@ -105,8 +105,8 @@
 // CHECK: }
 
 // CHECK: struct DifferentTypesArrayByVal {
-// CHECK:   subscript(x: Int32) -> Int32 { mutating get }
```

Comment:
I think this is a side effect of the way subscripts are printed, not a reflection of change in functionality? The check for "mutating" is [predicated on the subscript having a getter, and not a read accessor](https://github.com/apple/swift/blob/main/lib/AST/ASTPrinter.cpp#L1072).